buildSrc/src/main/java/com/osacky/flank/gradle/FladleConfig.kt

```diff
@@ -256,4 +256,44 @@ interface FladleConfig {
    */
   @get:Input
   val fullJunitResult: Property<Boolean>
+
+  /**
+   * A list of up to 100 additional APKs to install, in addition to those being directly tested.
+   * The path may be in the local filesystem or in Google Cloud Storage using gs:// notation.
+   */
+  @get:Input
+  @get:Optional
+  val additionalApks: ListProperty<String>
+
+  /**
+   * Enable using average time from previous tests duration when using SmartShard and tests did not run before.
+   * (default: false)
+   */
+  @get:Input
+  @get:Optional
+  val useAverageTestTimeForNewTests: Property<Boolean>
+
+  /**
+   * Set default test time expressed in seconds, used for calculating shards.
+   * (default: 120.0s)
+   */
+  @get:Input
+  @get:Optional
+  val defaultTestTime: Property<Double>
+
+  /**
+   * Set default parameterized class test time expressed in seconds, used for calculating shards.
+   * (default: 2x [defaultTestTime] => 240s)
+   */
+  @get:Input
+  @get:Optional
+  val defaultClassTestTime: Property<Double>
```

Comment:
can this be used in combination with `defaultTestTime`? what's the use case?

Reply:
> can this be used in combination with defaultTestTime?

Yes

> what's the use case?

Flank uses time from the previous run to divide tests into shards (smart flank feature). But when there is no previous result and/or the user don't want to use this feature default time is used. Until recently, default value was hidden from users but there was a feature request to make it possible to set desired value. That is for `defaultTestTime`
When it comes to `defaultClassTestTime`, logic is the same but for parameterized classes (flank is unable to shard tests from parameterized classes so it considers each class as one test -- therefore need for different default value)